### DISCUSSION TITLE
Stuck in a Boot Loop

### DISCUSSION THREAD

**Discussion by FreedomSevenOG** [dakanji/RefindPlus#154] [General]
[@Dayo](https://forums.macrumors.com/members/1159775/) Hey Dayi, I’ve got up to Testrefindplus and it’s on this screen(see attachment) after selecting option 1. For Mojave OS
It’s cycled around 3 times already
Meaning it goes through a reboot, shows option for OS, 1.Mojave boot into open core, sits with same screen and repeats
Is this correct ?

Just noticed I;the 4th cycle around the operating system says High Sierra, not Mojave
Yet Mojave IS the OS ?
I believe you mentioned in your write up something about havping an OS that was already patched
Maybe that is why it’s doing this ?
![IMG-5801](https://user-images.githubusercontent.com/126120602/221258771-ca5fc7dc-9892-47f3-80e3-a91ce333592f.JPG)
If so, what do I do now ?
Thanks

**Comment by FreedomSevenOG** [comment]
NVM I put dink in another comp, mounted Eli and deleted refine and open ore
Put back in Mac Pro and it’s booted fine
So just have to try process again

**Comment by dakanji** [comment]
Sorry, I don't understand your post.
If you have an issue to report about RefindPlus, then please use and follow the issue template.
